Classes/Fusion/Helper/NodeInfoHelper.php

```diff
@@ -30,6 +30,7 @@
 use Neos\Neos\Utility\NodeTypeWithFallbackProvider;
 
 /**
+ * @internal
  * @Flow\Scope("singleton")
  * @todo EEL helpers are still to be declared as internal
```

Comment:
```suggestion
```

This should go, since `NodeInfoHelper` is marked `@internal` now in L33.

Reply:
**UPDATE:** Just saw that #3711 contains all changes of #3703 and removes the `@todo` annotation. Maybe #3711 can just supersede #3703, so #3703 can be closed? wdyt?

Reply:
a i forgot how i did it, but yes this pr is basically step one and later we followup with https://github.com/neos/neos-ui/pull/3711 so ill merge it first ;)
Thanks for looking so deep into it :D